src/coreclr/vm/exceptmacros.h

```diff
@@ -240,7 +240,16 @@ VOID DECLSPEC_NORETURN RealCOMPlusThrowOM();
 
 #endif // !defined(FEATURE_EH_FUNCLETS)
 
-LONG WINAPI CLRVectoredExceptionHandler(PEXCEPTION_POINTERS pExceptionInfo);
+enum VEH_ACTION
```

Comment:
Do these really need to be in the header? Adding definitions to the header implies they are used in other places whereas this are not.

Reply:
Thanks @AaronRobinsonMSFT, it is actually used in two sources, the excep.cpp and excepx86.cpp. However, I have forgotten to modify the excepx86.cpp to get the result of the `CLRVectoredExceptionHandler` call into `VEH_ACTION`, it was still using `DWORD`. I am updating the change.